crabs/detection_tracking/train_model.py

```diff
@@ -111,6 +120,12 @@ def main(args) -> None:
         default="gpu",
         help="accelerator for pytorch lightning",
     )
+    parser.add_argument(
+        "--experiment_name",
+        type=str,
+        default="Sept2023",
+        help="the run name for mlflow for example the dataset used for the run",
```

Comment:
```suggestion
        help="the name for the experiment in MLflow, under which the current run will be logged. For example, the name of the dataset could be used, to group runs using the same data.",
```
Here the MLflow concepts for "run" and an "experiment" are a bit mixed up here, but this should make it clearer.